ROADMAP.md

```diff
@@ -0,0 +1,105 @@
+# AutoGluon Roadmap 2022
+
+In 2022, our top priorities are:
+
+- [ ] Windows OS Support
+- [ ] Python 3.9 Support
+- [ ] Time-Series Support
+- [ ] HuggingFace Integration
+- [ ] Improved Multi-modal Modeling
+- [ ] Advance SOTA in Stack Ensemble Research (Deeper Stack Ensembles)
+- [ ] Cloud Training & Deployment
+- [ ] Parallel Model Training
+- [ ] Parallel Hyperparameter Tuning
+- [ ] Distributed Model Training
+- [ ] Distributed Hyperparameter Tuning
+- [ ] Semi-supervised Learning
+- [ ] Enhanced Model Distillation
+- [ ] Online Inference Optimization
+- [ ] Meta-Learning
+- [ ] Improved Feature Type Inference
+- [ ] Co-variate Shift Detection
+- [ ] Co-variate Shift Correction
+- [ ] Exploratory Analysis
+- [ ] Model Interpretability
+- [ ] Model Uncertainty
+
+## AutoGluon 0.4 Release, Due: January 2022
+
+### Themes and Major Features
+
+- [x] Windows OS Support
+- [x] Python 3.9 Support
+- [ ] HuggingFace Integration
+- [ ] Torch Migration (Remove MXNet dependency)
+- [x] Parallel Model Training (2x training speed-up for bagging/stacking)
+- [x] Automated Feature Pruning/Selection
+- [x] Semi-supervised & Transductive Learning Support
+- [x] Automated Model Calibration via Temperature Scaling
+- [x] Cloud Training & Deployment Tutorials
+- [x] Feature Preprocessing Tutorial
+- [ ] Advanced Custom Model Tutorial
+- [x] Documentation Overhaul
+- [ ] Hyperparameter Tuning Overhaul
+- [x] Memory Usage Optimizations
+- [x] Various Performance Optimizations
+- [x] Various Bug Fixes
+
+# AutoGluon Roadmap 2021
+
+In 2021, our top priorities are:
+
+- [x] Make AutoGluon the most versatile AutoML framework via dedicated multi-modal image-text-tabular support ([paper](https://arxiv.org/abs/2111.02705)).
+- [x] Modularization of the various components of AutoGluon.
+- [x] Model Training Speed Optimizations.
+- [x] Model Inference Speed Optimizations.
+- [x] Model Quality Optimizations.
+- [x] Integration with [NVIDIA RAPIDS](https://developer.nvidia.com/rapids) for accelerated GPU training.
+- [x] Integration with [Intel sklearnex](https://github.com/intel/scikit-learn-intelex) for accelerated CPU training.
+- [x] Improved documentation and tutorials.
+
```

Comment:
+ - [x] Training and Inference containers

Reply:
added